src/themes/blueradiance/theme.css

```diff
@@ -261,6 +261,10 @@ html {
     background: rgba(0, 0, 0, 0.3);
 }
 
+.listItem:hover {
+    background: #10223e;
```

Comment:
I am not sure if it will look better, but we could change colors like this:
`focus`: `rgba(0, 0, 0, 0.5)`
`hover`: `rgba(0, 0, 0, 0.3)`
or
`focus`: `rgba(0, 0, 0, 0.3)`
`hover`: `rgba(0, 0, 0, 0.15)`

Reply:
I liked your second option!